champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/sword/FleshHook.java

```diff
@@ -200,7 +200,7 @@ private void shoot(Player player, ChargeData data, int level) {
         throwable.setCanHitFriendlies(true);
         championsManager.getThrowables().addThrowable(throwable);
 
-        VelocityData velocityData = new VelocityData(player.getLocation().getDirection(), 1 + data.getCharge(), false, 0, 0.2, 20, false);
+        VelocityData velocityData = new VelocityData(player.getLocation().getDirection(), getVelocityStrength(level) * data.getCharge(), false, 0, 0.2, 20, false);
```

Comment:
Since this used to have a minimum value of 1, im not sure this will be quite the same, maybe add them together instead of multiply

Reply:
Okay yeah, I wasn't sure what that 1 represented